minicbor-serde/src/ser.rs

```diff
@@ -35,14 +36,22 @@ impl<W: Write> Serializer<W> {
         &mut self.encoder
     }
 
+    #[must_use]
+    pub fn with_encode_unit_as_null(self, enable: bool) -> Self {
```

Comment:
For the public API I would prefer in place mutation and "serialize" instead of "encode", e.g. `pub fn serialize_unit_as_null(&mut self, val: bool) -> &mut Self`.

Reply:
done, I kept `with_serialize_unit_as_null` though, let me know if you prefer to have the method named `serialize_unit_as_null`

Reply:
I tweaked the names slightly in d31bf0f777661a7ec70dff239c5d42ee8a0142b7 and merged your PR.